common/content/buffer.js

```diff
@@ -415,6 +416,18 @@ const Buffer = Module("buffer", {
     set textZoom(value) { Buffer.setZoom(value, false); },
 
     /**
+     * @property {number} The current browser's zoom level, as a
+     *     percentage with 100 as 'normal'.
+     */
+    get zoomLevel() {
```

Comment:
I think it is set by `ZoomManager` this way, yes. It is possible to strip out the space, but if I recall correctly Pentadactyl also presents it with a leading space when under 100%. In my opinion it is not really an issue. But that is of course for the collaborators to decide.